Online-Messaging-App/src/app/home/profile/profile.component.html

```diff
@@ -0,0 +1,29 @@
+<div *ngIf="user" class="wrapper">
+    <mat-card>
+        <div class="icon">
+            <mat-card-header>
+                <mat-icon inline="true">account_box</mat-icon>
+                <b id="profileTitle">Profile</b>
+            </mat-card-header>
+        </div>
+        <mat-card-content>
+            <hr>
+            <br>
+            <b id="first">Username</b>
+            <mat-icon id="profilePicture" inline="true">person</mat-icon>
+            <br>
+            <p>{{user[0].username}}</p> <br>
```

Comment:
`<p>` tags break, so there should be a way you can re-format this whole block so that you don't have to have so many `<br>` tags. You can wrap a `<p>` tag in a `<b>` tag.

Reply:
done